store.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF8" />
+    <meta
+      name="viewport"
+      content="width=devicewidth, initialscale=1, shrinktofit=no"
+    />
+    <title>Karmastore</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="/css/store.css" />
+    <link rel="shortcut icon" type="image/xicon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <a class="logo" href="index.html"
+        ><img src="/img/karmalogo.svg" alt="karma logo"
+      /></a>
+      <nav>
+        <ul class="navlist">
+          <input type="checkbox" id="check" />
+          <div class="menu">
+            <li class="navitems">
+              <a class="navlinks" href="index.html">Meet Karma</a>
+            </li>
+            <li class="navitems">
+              <a class="navlinks" href="#">How it Works</a>
+            </li>
+            <li class="navitems">
+              <a class="navlinks" href="store.html">Store</a>
+            </li>
+            <li class="navitems"><a class="navlinks" href="#">Blog</a></li>
+            <li class="navitems"><a class="navlinks" href="#">Help</a></li>
+            <li class="navitems"><a class="navlinks" href="#">Login</a></li>
+          </div>
+          <label for="check" class="openmenu"
+            ><img src="/level3/menuhamburger.svg" alt="burger"
+          /></label>
+        </ul>
+      </nav>
+    </header>
+    <main class="storemain">
+      <section class="formsection">
+        <h1>Order your Karma wifi device today!</h1>
+
+        <form action="#">
+          <div class="fieldset">
```

Comment:
Calling this class fieldset is a bit confusing as the div isn't a fieldset. The [fieldset tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) is used to group several checkboxes or radio buttons together.